cosmos/settings.py

```diff
@@ -29,6 +29,7 @@
 enable_cache_profile = conf.getboolean("cosmos", "enable_cache_profile", fallback=True)
 dbt_profile_cache_dir_name = conf.get("cosmos", "profile_cache_dir_name", fallback="profile")
 virtualenv_max_retries_lock = conf.getint("cosmos", "virtualenv_max_retries_lock", fallback=120)
+default_copy_dbt_packages_value = conf.getboolean("cosmos", "default_copy_dbt_packages_value", fallback=False)
```

Comment:
Shall we document it here: https://github.com/astronomer/astronomer-cosmos/blob/main/docs/configuration/cosmos-conf.rst

Reply:
Yes, @pankajastro ! I'm creating a follow-up PR with documentation changes